### PR TITLE
VXFM-7352 wsman.first_sata_disk filters by using SATADOM model

### PR DESCRIPTION
**Pull request by danielseankim**

### CODE REVIEW

**Comment by danielseankim** [comment]
@gavin-scott can you merge this please?

File: lib/asm/wsman.rb [hunk 1]
```diff
@@ -1778,7 +1778,7 @@ def identify_chassis(options={})
                     :return_value => "0")
     end
 
-    # Get the FQDD of the first SATA Disk present on the node
+    # Get the FQDD of the first SATA Disk (SATADOM model) present on the node
```
Comment (gavin-scott) [inline lib/asm/wsman.rb]: Do you know for sure this method is only used to get the SATADOM? If so then this change seems fine.

Comment (danielseankim) [reply lib/asm/wsman.rb]: @gavin-scott from what I've heard, we are only supporting SATADOM, I believe. However, I can change this to run again with `if bus_protocol == 5` when `sata_disks.empty?` is true.

Comment (gavin-scott) [reply lib/asm/wsman.rb]: I looked through the code and the only spots calling it look like they are looking for satadom:

https://github.com/dell-asm/asm-deployer/blob/59cb2e910455c9490dc85592309aefca95d04fb2/lib/asm/provider/server/server.rb#L565
https://github.com/dell-asm/asm-deployer/blob/59cb2e910455c9490dc85592309aefca95d04fb2/lib/asm/provider/server/server.rb#L635

Might be nice to rename this to `satadom_disk` or something but this seems OK.

**Pull request merged**